sdk/eventhubs/azure-messaging-eventhubs/migration-guide.md

```diff
@@ -1,142 +1,148 @@
-# Guide to migrate from `azure-eventhubs` to `azure-messaging-eventhubs` (Event Hubs v3 to v5)
+# Guide for migrating to `azure-messaging-eventhubs` from `azure-eventhubs` and `azure-eventhubs-eph`
 
-This document is intended for users that are familiar with v3 of the Java SDK for Event Hubs library
-([`azure-eventhubs`][azure-eventhubs] and [`azure-eventhubs-eph`][azure-eventhubs-eph]) and wish to migrate their
-application to V5 of the same library.
+This guide assists in the migration to `azure-messaging-eventhubs` from `azure-eventhubs` and
```

Comment:
Based on template https://github.com/Azure/azure-sdk/blob/f4dd2025c87f6f4ce615ae4687d2ba527381137e/docs/policies/migration-guide-template.md

`This guide assists` -> `This guide is intended to assist`

Reply:
There are multiple libraries so I felt it was grammatically correct.